plugins/PostProcessingPlugin/scripts/PrintSkewCompensation.py

```diff
@@ -0,0 +1,405 @@
+"""
+By GregValiant (Greg Foresi) in April of 2025.
+Print Skew Compensation is designed to use either Cura post processing, Marlin M852, or Klipper SET_SKEW to compensate for hardware skewing of prints.
+
+The measurements for skew are different for each printer.  A log file will be created within the configuration folder 'scripts' sub-folder and will be based on the printer name (Ex: Ender-3Pro.skew.log).  When you change printers a new log file will be created but it is up to the user to make sure the settings are correct for that specific printer.
+
+Skew Calibration model files are available on my Git page at:
+https://github.com/GregValiant/GV-PostProcessors/tree/StartPoint/Skew%20Calibration%20Models
+
+"""
+
+from UM.Application import Application
+from ..Script import Script
+from UM.Message import Message
+from UM.Logger import Logger
+import re
+import math
+import os.path
+from UM.Resources import Resources
+import os
+
+class PrintSkewCompensation(Script):
+    def initialize(self) -> None:
+        super().initialize()
+
+        # Get the "Printer Name.skew.log" file and parse it for the settings.  If there isn't a log file, then create it.  The log file gets updated to the current settings every time the script runs.
+        set_lines = None
+        active_printer = Application.getInstance().getGlobalContainerStack().getProperty("machine_name", "value")
+        try:
+            config_path = Resources.getConfigStoragePath()
+            
+            pp_path = config_path + "\\scripts\\"
```

Comment:
```suggestion
            pp_path = os.path.join(config_path, "scripts")
```

Different operating systems handle the path joining differently so it's easier to use os.path.join to merge paths

Reply:
Later in execute I use:
log_file_name = config_path + "\\scripts\\" + active_printer + ".skew.log"
If I do the same trick as you suggest - how do I concatenate the file name?

Reply:
That would be ```log_file_name = os.path.join(config_path, "scripts", f"{active_printer}.skew.log")```

Reply:
Alternatively if you store the path in self for the pp_path, self.pp_path = os.path.join(config_path, "scripts")
then later you can just use that:
```log_file_name = os.path.join(self.pp_path, f"{active_printer}.skew.log")```